doc/introduction/interfaces.rst

```diff
@@ -408,81 +427,111 @@ At the moment, it takes into account the following parameters:
++                  +-------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+-------------+-------------+
+|                  | ``"backprop"``                |    :rd:`1`   |      :rd:`1`  |   :rd:`1`    | :rd:`9`      |  :rd:`1`      | :rd:`1`        | :rd:`1`        | :rd:`1`     | :rd:`1`     | :rd:`1`     |
++                  +-------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+-------------+-------------+
+|                  | ``"adjoint"``                 |    :rd:`2`   |     :rd:`2`   |    :rd:`2`   | :rd:`9`      | :rd:`2`       |   :rd:`2`      | :rd:`2`        | :rd:`2`     | :rd:`2`     | :rd:`2`     |
++                  +-------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+-------------+-------------+
+|                  | ``"parameter-shift"``         |    :rd:`2`   |     :rd:`2`   |    :rd:`2`   | :rd:`9`      | :rd:`2`       |   :rd:`2`      | :rd:`2`        | :rd:`2`     | :rd:`2`     | :rd:`2`     |
++                  +-------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+-------------+-------------+
+|                  | ``"finite-diff"``             |    :rd:`2`   |     :rd:`2`   |    :rd:`2`   | :rd:`9`      | :rd:`2`       |   :rd:`2`      | :rd:`2`        | :rd:`2`     | :rd:`2`     | :rd:`2`     |
++                  +-------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+-------------+-------------+
+|                  | ``"spsa"``                    |    :rd:`2`   |     :rd:`2`   |    :rd:`2`   | :rd:`9`      | :rd:`2`       |   :rd:`2`      | :rd:`2`        | :rd:`2`     | :rd:`2`     | :rd:`2`     |
++                  +-------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+-------------+-------------+
+|                  | ``"hadamard"``                |    :rd:`2`   |     :rd:`2`   |    :rd:`2`   | :rd:`9`      | :rd:`2`       |   :rd:`2`      | :rd:`2`        | :rd:`2`     | :rd:`2`     | :rd:`2`     |
++                  +-------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+-------------+-------------+
+|                  | ``"reversed-hadamard"``       |    :rd:`2`   |     :rd:`2`   |    :rd:`2`   | :rd:`9`      | :rd:`2`       |   :rd:`2`      | :rd:`2`        | :rd:`2`     | :rd:`2`     | :rd:`2`     |
```

Comment:
@albi3ro are these newly added entries accurate? I just copy-pasted the table values in the "hadamard" row

Reply:
So "hadamard" supports probs, but reversed, direct, and revsed-direct do not.

Reply:
Is that the only discrepancy?

Reply:
I think so.

Reply:
The conversation seems concluded. So resolved.